lib/matplotlib/axes/_axes.py

```diff
@@ -2291,6 +2291,14 @@ def bar(self, x, height, width=0.8, bottom=None, *, align="center",
         xerr = kwargs.pop('xerr', None)
         yerr = kwargs.pop('yerr', None)
         error_kw = kwargs.pop('error_kw', {})
+        ezorder = error_kw.pop('zorder', None)
+        if ezorder is None:
+            ezorder = kwargs.get('zorder', None)
+            if ezorder is not None:
+                # If using the bar zorder, increment slightly to make sure
+                # errorbars are drawn on top of bars
+                ezorder += 0.1
```

Comment:
Not sure it matters too much, but for inset indicators and ticks, the default difference is only 0.01.

Reply:
👍 no harm in keeping things consistent, I'll change it.